tendrl/alerting/objects/alert/__init__.py

```diff
@@ -0,0 +1,125 @@
+from tendrl.alerting.objects \
+    import AlertingBaseObject
+from tendrl.commons.etcdobj import EtcdObj
+from tendrl.commons.singleton import to_singleton
+
+alert_severity_map = {
+    'INFO': 0,
+    'WARNING': 1,
+    'CRITICAL': 2
+}
+
+
+class Alert(AlertingBaseObject):
+    def __init__(
+        self,
+        alert_id,
+        node_id=None,
+        time_stamp=None,
+        resource=None,
+        current_value=None,
+        tags=None,
+        alert_type=None,
+        severity=None,
+        significance=None,
+        ackedby=None,
+        acked=None,
+        pid=None,
+        source=None,
+        *args,
+        **kwargs
+    ):
+        super(Alert, self).__init__(*args, **kwargs)
+        self.alert_id = alert_id
+        self.node_id = node_id
+        self.time_stamp = time_stamp
+        self.resource = resource
+        self.current_value = current_value
+        self.tags = tags
+        self.alert_type = alert_type
+        self.severity = severity
+        self.significance = significance
+        self.ackedby = ackedby
+        self.acked = acked
+        self.pid = pid
+        self.source = source
+
+
+class _Alert(EtcdObj):
+    """A table of the node context, lazily updated
+
+    """
+    __name__ = 'alerting/alerts/%s'
+    _tendrl_cls = Alert
+
+    def render(self):
+        self.__name__ = self.__name__ % self.alert_id
+        return super(_Alert, self).render()
+
+
+@to_singleton
+class AlertUtils(object):
+    def __init__(self):
+        self.etcd_client = tendrl_ns.etcd_orm.client
```

Comment:
Why to maintain this as member variable? Cant we access as when required using `tendrl_ns.etcd_orm` ?